card/mod/02_basic_types/spec/set/type/pointer_spec.rb

```diff
@@ -50,6 +50,93 @@
   end
 
 
+  let(:pointer) do
+    Card.create name: 'tp', type: 'pointer',
+                content: "[[item1]]\n[[item2]]"
+  end
+
+  describe '#added_item_names' do
+    it 'recognizes added items' do
+      Card::Auth.as_bot do
+        pointer
+        in_phase before: :approve,
+                 on: :save,
+                 trigger: ->{
+                   pointer.update_attributes!(
+                     content: "[[item1]]\n[[item2]]\n[[item3]]"
+                  )
+                } do
+          expect(added_item_names).to eq ['item3']
+        end
+      end
+    end
+
+    it 'ignores order' do
+      Card::Auth.as_bot do
+        pointer
+        in_phase before: :approve,
+                 on: :save,
+                 trigger: ->{
+                   pointer.update_attributes!(
+                     content: "[[item2]]\n[[item1]]"
+                   )
+                 } do
+          expect(added_item_names).to eq []
+        end
+      end
+    end
+  end
+
+  describe '#dropped_item_names' do
+    it 'recognizes dropped items' do
+      Card::Auth.as_bot do
+        pointer
+        in_phase before: :approve,
+                 on: :save,
+                 trigger: ->{
+                   pointer.update_attributes!(
+                     content: "[[item1]]"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.